packages/api-contract/src/types.ts

```diff
@@ -55,8 +55,8 @@ export interface InterfaceContractCalls {
 
 export interface ContractCallOutcome {
   debugMessage: Text;
-  gasConsumed: u64;
-  gasRequired: u64;
+  gasConsumed: u64 | WeightV2;
```

Comment:
This should just be `Weight` - this is a bit tricky, but the runtime will inject the correct type.

Reply:
Changed